presto-main/src/main/resources/webapp/src/components/LivePlan.jsx

```diff
@@ -247,12 +247,28 @@ export class LivePlan extends React.Component<LivePlanProps, LivePlanState> {
             this.updateD3Stage(stage, graph);
         });
 
-        this.state.render(d3.select("#plan-canvas g"), graph);
+        const inner = d3.select("#plan-canvas g");
+        this.state.render(inner, graph);
 
         const svg = this.state.svg;
         svg.selectAll("g.cluster").on("click", LivePlan.handleStageClick);
-        svg.attr("height", graph.graph().height);
-        svg.attr("width", graph.graph().width);
+
+
+        const graphWidth = graph.graph().width + 100;
+        const graphHeight = graph.graph().height + 100;
+        const width = parseInt(window.getComputedStyle(document.getElementById("live-plan"), null).getPropertyValue("width").replace(/px/, "")) - 50;
```

Comment:
These are a little hard to read. I'd, at a minimum, pull out `window.getComputedStyle(document.getElementById("live-plan"), null)` into a variable since it's used in both lines.

Reply:
Also is the `null` in `window.getComputedStyle(document.getElementById("live-plan"), null)` required? Online docs seem to indicate that omitting is is equivalent to passing `null`.